Description: Disable test
 tests/basic.rs wants to pull container image nitrokey/nethsm. Since there's no
 internet access during for buildds, and the rustainers dev dependency is not
 available, disable the test.
Author: Tobias Deiminger <tobias.deiminger@posteo.de>
Last-Update: 2025-09-08 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: nethsm-sdk-rs/Cargo.toml
===================================================================
--- nethsm-sdk-rs.orig/Cargo.toml
+++ nethsm-sdk-rs/Cargo.toml
@@ -39,7 +39,7 @@ repository = "https://github.com/Nitroke
 name = "nethsm_sdk_rs"
 path = "src/lib.rs"
 
-[[test]]
+[[disabled.test]]
 name = "basic"
 path = "tests/basic.rs"
 
